src/component/LogPages/ForgetPwdPage/ForgetPwdPage.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import LogPagesProps from "../LogPagesProps/LogPagesProps";
+import "./ForgetPwdPage.scss";
+import emailIcon from "../../img/emailIcon.svg";
+
+export default function ForgetPwdPage({ active }) {
+  return (
+    <LogPagesProps active={active} header="Find your password" footer="">
+      <form>
+       
+          
+       <div className="forgetPwdPage-input">
+       <img src={emailIcon}></img>
+       <input
+          type="text"
+          id="forgetPwdPage-email"
+          name="forgetPwdPage-email"
+          placeholder="Please input your Email address"
+        ></input>
```

Comment:
recommend using the self-closing tag in this situation as Airbnb-style required:
`<Foo variant="stuff"></Foo>`  
to 
`<Foo variant="stuff" />`

[link](https://airbnb.io/javascript/react/#tags)